src/components/SignerPanel/SigningStatus.js

```diff
@@ -110,64 +107,85 @@ class SigningStatus extends React.Component {
     )
   }
   render() {
-    const { status } = this.props
+    const { theme, status } = this.props
     return (
-      <React.Fragment>
-        <Status>
+      <div>
+        <Status
+          color={theme.feedbackContent}
+          background={theme.feedbackSurface}
+        >
           <StatusImage status={status} />
-          <p>{this.getLabel()}</p>
+          <p
+            css={`
+              margin-top: ${3.5 * GU}px;
+              ${textStyle('body2')};
+            `}
+          >
+            {this.getLabel()}
+          </p>
         </Status>
-        <AdditionalInfo>{this.getInfo()}</AdditionalInfo>
+        <Info>{this.getInfo()}</Info>
         {this.getCloseButton()}
-      </React.Fragment>
+      </div>
     )
   }
 }
 
 const Status = styled.div`
-  margin-top: 80px;
-  margin-bottom: 40px;
+  background: ${({ background }) => background};
+  height: 360px;
+  margin-bottom: ${3 * GU}px;
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  color: ${theme.textSecondary};
-  img {
-    margin-bottom: 20px;
-  }
+  color: ${({ color }) => color};
 `
 
-const AdditionalInfo = styled(Info)`
-  p + p {
-    margin-top: 10px;
-  }
-`
+const StatusImage = ({ status }) => {
+  const theme = useTheme()
+  const color = isSignatureError(status)
+    ? theme.negative
+    : isSignatureSuccess(status)
+    ? theme.positive
+    : theme.feedbackSurfaceContentSecondary
+
+  return (
+    <div
+      css={`
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        width: 60px;
+        height: 60px;
+        border: 2px solid ${color};
```

Comment:
We should increase the border when it's successful or had an error (designs have them at 3px but we may want to play around with this).

Reply:
If I recall correctly this was updated to `40px` `width` and `height` and `3px` border right?

Reply:
We have to check with @dizzypaty on this; that was only for the Voting app but may not be the case for the signing panel itself (since it's feedback surface is much larger).

Reply:
Yes, it was updated in the Voting app module (to be 40x40px) to make it more compact for that context. But we should keep the current size (60x60px) on the signer panel. As Brett said, the reason being its surface being larger here and being the most important item the users need to look at when performing this action. Once we standarize this component, it will make sense to define all its possible sizes depending on where is presented.